dogs/src/app/components/breed-page/breed-page.component.ts

```diff
@@ -16,15 +16,27 @@ export class BreedPageComponent implements OnInit {
 
 	public constructor(
 		private dogService: DogService,
-		private activatedRoute: ActivatedRoute
+		private activatedRoute: ActivatedRoute,
 	) { }
 
 	public ngOnInit(): void {
+		this.getBreedNameFromURL();
+		this.getBreedImage();
+		this.getSubBreedNames
+	}
+
+	private getBreedNameFromURL(): void {
```

Comment:
```suggestion
	private prepareBreedNameFromURL(): void {
```
sorry, my bad
let's name with "prepare" prefix, because the method does not return anything